cmake/config/CMakeLists.txt

```diff
@@ -219,4 +225,9 @@ FOREACH(_name ${_deal_ii_features_sorted})
   ENDFOREACH()
 ENDFOREACH()
 
+# just for mac and linux
+if(NOT MSVC)
+  add_subdirectory(pkgconfig)
+endif()
```

Comment:
Nothing in the subdirectory `pkgconfig` requires Linux or Mac - granted the configuration you create might be useless, but that doesn't warrant this special casing. Can you remove this `if` guard and refactor the whole subdirectory into this one, please?

Reply:
yes. I used `/` as folder separator, that's why I used that guard. Should I move the contents of `pkgconfig/CMakeLists.txt` into this one or I should create a function/macro that I source?